### PR TITLE
Add more information about the workshops

### PR DESCRIPTION
**Pull request by bvdmitri**
## Briefly Describe your changes

This PR adds more information about the upcoming workshops, such as location, av info, catering info, etc

## Checklist for merge
- [x] I built the website locally and tested it using `Franklin.serve()`
- [x] All CI checks have passed and you have tested the online version (click on the list of checks and click `Build and Deploy / Preview` to see the preview)
- [ ] Somebody else reviewed my changes (use your best judgement if you need to merge quickly

## After merge
- [ ] Closed relevant issues

### CODE REVIEW

File: 2024/workshops.md [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
-**Note!**: This list is preliminary and contains only the workshops that have been confirmed so far. A few more may appear in the upcoming days.
 
-Checkout the full [schedule on PreTalx](https://pretalx.com/juliacon2024/schedule/), or click [here](https://discourse.julialang.org/t/juliacon-2024-schedule-is-now-out/113717/3) for the rest of the announcement and Important FAQs!
+## Practical Information
+
+- **What:** Workshops day ahead of JuliaCon
+- **Date:** July 9, 2024
+- **Time:** Walk-in at 8:30, starts at 9:00, ends at 17:00
+- **Location:** Neuron building, Eindhoven University of Technology campus
+- **Tickets:** Yes, participants need a ticket
```
Comment (lucaferranti) [inline 2024/workshops.md:10]:
```suggestion
- **Tickets:** Yes, participants need a ticket. Get yours [here](https://juliacon.org/2024/tickets/)
```

File: 2024/workshops.md [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
+- 15:00 - Coffee moment
+- 16:30 - ‘Worstenbroodje’ snack.
+
+*All coffee moments also include tea, water & some kind of sweet/snack etc.*
+
+## Program & workshops descriptions
+
+Checkout the full [schedule on PreTalx](https://pretalx.com/juliacon2024/schedule/), or click [here](https://discourse.julialang.org/t/juliacon-2024-schedule-is-now-out/113717/3) for the rest of the announcement and Important FAQs!
+
+### Building REST APIs in Julia
```
Comment (lucaferranti) [inline 2024/workshops.md:50]: while you're at it, this was actually withdrawn, so could be removed from the schedule

**Review by lucaferranti — APPROVED** [review]
one small suggestion and a comment, otherwise LGTM

File: 2024/workshops.md (hunk 2 above)
Comment (bvdmitri) [reply 2024/workshops.md:50]: hm, do we have a document with the latest info regarding the confirmed workshops?

**Pull request merged**